src/ansys/fluent/core/__init__.py

```diff
@@ -44,44 +44,16 @@ def version_info() -> str:
     return _VERSION_INFO if _VERSION_INFO is not None else __version__
 
 
-def set_log_level(level: Any) -> None:
-    """Set logging level.
+file_path = os.path.abspath(__file__)
+file_dir = os.path.dirname(file_path)
+yaml_path = os.path.join(file_dir, "logging_config.yaml")
 
-    Parameters
-    ----------
-    level : Any
-        Any of the logging level (CRITICAL, ERROR, WARNING, INFO, DEBUG)
-        in string or enum format
-    """
-    LOG.set_level(level)
-
-
-def enable_logging_to_stdout() -> None:
-    """Enable logging to stdout."""
-    LOG.enable_logging_to_stdout()
-
-
-def disable_logging_to_stdout() -> None:
-    """Disable logging to stdout."""
-    LOG.disable_logging_to_stdout()
-
-
-def enable_logging_to_file(filepath: Optional[str] = None) -> None:
-    """Enable logging to file.
-
-    Parameters
-    ----------
-    filepath : str, optional
-        filapath, a default filepath will be chosen if filepath is not
-        passed
-    """
-    LOG.enable_logging_to_file(filepath)
-
-
-def disable_logging_to_file() -> None:
-    """Disable logging to file."""
-    LOG.disable_logging_to_file()
+# Load the logging configuration from a YAML file
+with open(yaml_path, "rt") as f:
+    config = yaml.safe_load(f)
 
+# Configure the logging system
+logging.config.dictConfig(config)
```

Comment:
@alikasim72 I looked into the question of what happens if this method is called more than once (we are just one library):
https://stackoverflow.com/a/47496535

So, different configs cannot play together?